build-logic/publishing/build.gradle.kts

```diff
@@ -12,3 +14,9 @@ dependencies {
     implementation("com.github.johnrengelman.shadow:com.github.johnrengelman.shadow.gradle.plugin:7.0.0")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin")
 }
+
+tasks.withType<KotlinCompile>().configureEach {
+    kotlinOptions {
+        jvmTarget = "11"
```

Comment:
Is it possible to manage this version in the gradle properties file?

Reply:
I don't think so, but feel free to improve it